### ISSUE TITLE
subcommand to just trigger replication (without creating snapshots)

### ISSUE THREAD

**Issue by problame** [zrepl/zrepl#85]
would be useful for debugging connectivity problems
after #78

**Comment by shivno** [comment]
maybe a --dry-run option or similar for everything would be better (if possible)

**Comment by problame** [comment]
the *trigger* part has been implemented in 0.1 using the `zrepl signal wakeup JOB` subcommand.
Dry run has been made a separate issue.
